cryptofeed/feed.py

```diff
@@ -4,21 +4,27 @@
 Please see the LICENSE file for the terms and conditions
 associated with this software.
 '''
+import os
+import yaml
 import uuid
 from collections import defaultdict
 
 from cryptofeed.callback import Callback
-from cryptofeed.defines import (ASK, BID, BOOK_DELTA, FUNDING, L2_BOOK, L3_BOOK,
-                                LIQUIDATIONS, OPEN_INTEREST, TICKER, TRADES, VOLUME)
+from cryptofeed.defines import (ASK, BID, BOOK_DELTA, FUNDING, L2_BOOK, L3_BOOK, LIQUIDATIONS,
+                                OPEN_INTEREST, PROFILE, TICKER, TRADES, TRANSACTIONS, VOLUME)
 from cryptofeed.exceptions import BidAskOverlapping, UnsupportedDataFeed
 from cryptofeed.standards import feed_to_exchange, get_exchange_info, load_exchange_pair_mapping, pair_std_to_exchange
 from cryptofeed.util.book import book_delta, depth
 
+path = os.path.dirname(os.path.abspath(__file__))
```

Comment:
constant type defs at the file level like this should be in all caps

Reply:
@bmoscon, this is part of the 'config file' implementation topic: I comment on this topic below.